packages/babel-preset/esm.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+const envPreset = require('@babel/preset-env');
+const { loadConfig } = require('browserslist');
+const modulesPreset = require('@babel/preset-modules');
+const reactPreset = require('@babel/preset-react');
+const plugins = require('./plugins');
+
+function getTargets({ configPath, ignoreBrowserslistConfig }) {
+  if (
+    ignoreBrowserslistConfig ||
+    !loadConfig({ path: path.resolve(configPath) })
+  ) {
+    return { esmodules: true };
+  }
+
+  // nothing so preset env looks for the browserslist config
+  return undefined;
+}
+
+/**
+ * This is an experimental preset that uses the leaner preset-modules. It's
+ * seperate from the other one for quicker iteration and to make more clear it's
+ * not stable
+ */
+function preset(api, options = {}) {
+  const {
+    runtime,
+    useBuiltIns,
+    debug = true,
```

Comment:
should we really be defaulting `debug` to `true`? it's `false` in the other config

Reply:
We shouldn't oops